### PR TITLE
Apply i18n-extract-server and i18n-extract-merge and translate missin…

### PR DESCRIPTION
**Pull request by larkox**
#### Summary
Apply i18n-extract-server and i18n-extract-merge and translate missing strings

#### Ticket Link
None

### CODE REVIEW

**Comment by larkox** [comment]
@levb regarding your comment here: https://github.com/mattermost/mattermost-plugin-apps/pull/273#issuecomment-958155087

The correct make command are `make i18n-extract-server` and `make i18n-merge-server`. I fixed it here, and added the missing spanish translations.

@hanzei It would be awesome if we could add these to the ci. I think it would still fail on master, since `make i18n-merge-server` removes the "line break at the end of the file" that my IDE adds automatically, but that is something that can be solved easily. Nevertheless, I would say it is really low priority.

**Review by hanzei — APPROVED** [review]

**Review by hanzei — CHANGES_REQUESTED** [review]
Actually, when I run ` make i18n-extract-server` locally, your changes get reverted. Are you using `v2.1.2`?

**Comment by larkox** [comment]
@hanzei Weird... it seems that I have indeed v2.1.2. When I run i18n-extract-server the only change I get is the line break at the end of the file. What changes are you seeing?

**Review by levb — APPROVED** [review]
TY!

**Comment by hanzei** [comment]
On `master`:

```
$ make i18n-extract-server
Please update your assets/i18n/translate.*.json files and execute "make i18n-merge-server"
If you don't want to change any locale file, simply remove the assets/i18n/translate.??.json file before calling "make i18n-merge-server"
If you want to add a new language (for example french), simply run "touch assets/i18n/active.fr.json" and then run the "make i18n-extract-server" again
[sumacheb@jupiter apps]$ git diff 
─────────────────────────────────────────────────────────────────────────────────────────────────────────
modified: assets/i18n/active.en.json
─────────────────────────────────────────────────────────────────────────────────────────────────────────
@ assets/i18n/active.en.json:13 @
  "command.debug.kv.clean.hint": "[ App ID ]",
  "command.debug.kv.clean.label": "clean",
  "command.debug.kv.clean.submit": "Deleted {{.Count}} keys for `{{.AppID}}`, namespace `{{.Namespace}}`.",
  "command.debug.kv.create.description": "Create a new key-value for an App.",
  "command.debug.kv.create.hint": "[ AppID keyspec ]",
  "command.debug.kv.create.label": "create",
  "command.debug.kv.description": "View and update apps' KV stores.",
  "command.debug.kv.edit.description": "View or edit specific KV keys of an app.",
  "command.debug.kv.edit.hint": "[ AppID keyspec ]",
```

File: server/builtin/debug_kv_edit_modal.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ func (a *builtinApp) debugKVEditModal() handler {
 					a.conf.I18N().LocalizeWithConfig(loc, &i18n.LocalizeConfig{
 						DefaultMessage: &i18n.Message{
 							ID:    "modal.kv.edit.submit.stored",
-							Other: "Stored:\n```\nKey: {{.Key}}\n\n{{.Value}}\n```\n",
+							Other: "Stored:\\n```\\nKey: {{.Key}}\\n\\n{{.Value}}\\n```\\n",
 						},
```
Comment (hanzei) [inline server/builtin/debug_kv_edit_modal.go:119]: Why are escaped slashes needed here?

**Review by hanzei — CHANGES_REQUESTED** [review]
I'm confused. The CI check is green on `master`. Why is this PR needed?

**Pull request closed without merging**